.circleci/config.yml

```diff
@@ -262,6 +269,44 @@ defaults:
 # -----------------------------------------------------------------------------------------------
 jobs:
 
+  b_win:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run: systeminfo
+      - run:
+          name: "Installing dependencies"
+          command: .\.circleci\win_install_dependencies.ps1
+      - run:
+          name: "Building Solidity"
+          command: .\scripts\ci\build_windows.ps1 Debug
+      - run:
+          name: "Running soltest"
+          command: .\build\test\Debug\soltest.exe --show-progress -- --testpath .\test --no-smt --evmonepath=evmone\bin\evmone.dll
+
+  b_win_release:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run: systeminfo
+      - run:
+          name: "Installing dependencies"
+          command: .\.circleci\win_install_dependencies.ps1
+      - run:
+          name: "Building Solidity"
+          command: .\scripts\ci\build_windows.ps1 Release
```

Comment:
The CMAKE_BUILD_TYPE does not matter too much. What matters here is that we have to create an empty prerelease.txt file.